pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitFinalScanUseCase.java

```diff
@@ -151,7 +151,7 @@ public Optional<ScanError> processRGPFinalScan(ScanPayload scanPayload) {
                  return result;
              }
             KitRequestShipping rgpRnaKit =
-                    subkits.stream().filter(subkit -> subkit.getKitTypeName().contains("RNA")).findFirst().orElseThrow();
+                    subkits.stream().filter(subkit -> subkit.getKitTypeName().equals("RNA")).findFirst().orElseThrow();
```

Comment:
If `subkit` is null or `kit_type_name` is null, this will NPE.  A yoda statement like "RNA".equals(...) will save you from an NPE if kit_type_name is null (but not subkit).